core/src/ast.rs

```diff
@@ -157,39 +133,42 @@ pub enum Jump<T> {
     Jgz(Node<ValueSource<T>, T>),
 }
 
-/// All types unique to the source AST live here
+/// All types unique to the source AST live here.
```

Comment:
lowkey dont like the period but im cool if you have a reason for it

Reply:
looked weird without it in the doc browser